lib/diagnostic.rb

```diff
@@ -91,7 +91,7 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.delete something, running out of time!'
 
```

Comment:
you could use delete, among many other things. one of the simplest answers would be 

`arr.pop 2`